base_layer/core/src/validation/block_validators.rs

```diff
@@ -108,12 +114,18 @@ impl<B: BlockchainBackend> Validation<Block, B> for FullConsensusValidator {
         trace!(target: LOG_TARGET, "FCV - Coinbase output ok for {}", &block_id);
         check_block_weight(block, &self.rules.consensus_constants())?;
         trace!(target: LOG_TARGET, "FCV - Block weight ok for {}", &block_id);
+        check_duplicate_transactions_inputs(block)?;
```

Comment:
Can you add the description of the check to the validate function doc string list.

Reply:
Done